assets/css/properties_panel/_vehicle_properties_panel.scss

```diff
@@ -1,3 +1,4 @@
+$vpp-header-z-index: 10;
```

Comment:
I'm slightly confused on the naming of this variable - it sounds like there VPP header should have this particular z-index, but as far as I can tell it doesn't, so I'm having a little bit of trouble wrapping my head around what it's supposed to mean. Apologies if I'm missing something obvious.

Reply:
That is absolutely fair! Playing around with some different options to make that more obvious.

Reply:
@lemald in thinking about z-index variable organization, I tried a different approach for organizing z-index variables by context inspired by [this post ](https://www.smashingmagazine.com/2021/02/css-z-index-large-projects/). It is a variation on some of the recommendations - instead of using naming conventions to indicate the context, I found it personally helpful to represent each context as a map so it is a bit easier to identify all the defined z-indexes within a single context. Since we are using pre-existing z-indexes it felt clearer to me to just use the existing values rather than try to make each value relative to eachother.

This change is a bit tangential to the PR - I could alternatively add a $properties-panel-header-z-index variable `app.scss` to follow the existing pattern.

Reply:
Okay, I think this looks good! I share @firestack's question though and I want to give this a second look tomorrow just to make sure that everything adds up with the z-indexes given how tricky this all is.

Reply:
Thanks! Addressed that comment [here](https://github.com/mbta/skate/pull/1773#discussion_r1008055508) and removed that line